applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -305,41 +274,32 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
     }
 
     template <typename FilteredElementType>
-    bool check_pipe_equilibrium(const FilteredElementType& open_pipe_elements, double amax, unsigned int mPipingIterations)
+    bool CheckPipeEquilibrium(const FilteredElementType& rOpenPipeElements, double MaxPipeHeight, unsigned int MaxNumberOfPipingIterations)
     {
-        bool         equilibrium = false;
-        bool         converged   = true;
-        unsigned int PipeIter    = 0;
-        // todo: JDN (20220817) : grow not used.
-        // bool grow = true;
+        bool         equilibrium      = false;
+        bool         converged        = true;
+        unsigned int piping_iteration = 0;
 
         // calculate max pipe height and pipe increment
-        double da = CalculatePipeHeightIncrement(amax, mPipingIterations);
+        double da = CalculatePipeHeightIncrement(MaxPipeHeight, MaxNumberOfPipingIterations);
 
-        while (PipeIter < mPipingIterations && !equilibrium && converged) {
-            // set equilibrium on true
+        while (piping_iteration < MaxNumberOfPipingIterations && !equilibrium && converged) {
             equilibrium = true;
 
             // perform a flow calculation and stop growing if the calculation doesn't converge
             converged = this->Recalculate();
 
-            // todo: JDN (20220817) : grow not used.
-            // if (!converged)
-            //{
-            //    grow = false;
-            //}
-
             if (converged) {
                 // Update depth of open piping Elements
                 equilibrium = true;
-                for (auto OpenPipeElement : open_pipe_elements) {
+                for (auto OpenPipeElement : rOpenPipeElements) {
```

Comment:
To comply with the Kratos Style Guide:
```suggestion
                for (auto p_open_pipe_element : rOpenPipeElements) {
```

Reply:
done